src/Sentry/Platforms/Android/NdkHandlerStrategy.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Sentry
+{
```

Comment:
We prefer File-scoped namespace declaration over Block-scoped namespace declaration in this repo.